ext/LinearSolveEnzymeExt.jl

```diff
@@ -4,11 +4,13 @@ using LinearSolve
 using LinearSolve.LinearAlgebra
 isdefined(Base, :get_extension) ? (import Enzyme) : (import ..Enzyme)
 
-
 using Enzyme
 
 using EnzymeCore
 
+@inline EnzymeCore.EnzymeRules.inactive_type(v::Type{LinearSolve.KrylovJL}) = true
```

Comment:
probably add a minimum version 0.6.0 on EnzymeCore ?

Reply:
That's already the minimum